src/settings/Settings.js

```diff
@@ -213,11 +213,16 @@ export const SETTINGS = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
         default: "en",
     },
-    "analyticsOptOut": {
+    "analyticsOptIn": {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
-        displayName: _td('Opt out of analytics'),
+        displayName: _td('Send analytics data'),
         default: false,
     },
+    "showCookieBar": {
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
+        displayName: _td('Show cookie bar'),
+        default: true,
+    },
```

Comment:
Will this give us a separate settings toggle saying, "Show cookies bar"? Either way, don't we just need to show it if `analyticsOptIn === null`?